### PR TITLE
Full E2E test of AI proxy in Playwright and NEAR sandbox

### PR DESCRIPTION
**Pull request by petersalomonsen**
Test the AI proxy in Playwright connecting to the Fungible Token contract in a NEAR sandbox.

```mermaid
graph TD
    A[AI Chat Web Page] -->|Connects with Conversation ID| B[AI Proxy Server]
    B -->|Asks for Balance with Hashed Conversation ID| C[NEAR Fungible Token Smart Contract]
    B -->|Connects if Balance is Sufficient| D[OpenAI Mock Server]
    subgraph E[NEAR Sandbox]
        C
    end
```

https://github.com/user-attachments/assets/031b1df7-f614-40d1-a7d6-7e3ec8f72cba

### CODE REVIEW

**Comment by petersalomonsen** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@petersalomonsen Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Weekly streak is on the road, smart strategy! Secure your streak with another PR!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=petersalomonsen/quickjs-rust-near/25&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/petersalomonsen?type=bot&pr=petersalomonsen/quickjs-rust-near/25&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/petersalomonsen)





<details><summary>Current status: <i>executed</i></summary>

| Reviewer | Score |
|--------|--------|
| @Tguntenaar  | 8 |


Your contribution is much appreciated with a final score of 8!
You have received 153 (80 base + 25 weekly bonus + 40 monthly bonus + 5% lifetime bonus) Sloth points for this contribution

@Tguntenaar received 25 Sloth Points for reviewing and scoring this pull request.

Congratulations @petersalomonsen! Your PR was highly scored and you completed another monthly streak! To keep your monthly streak make another pull request next month and get 8+ score for it
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/petersalomonsen)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Review by Tguntenaar — APPROVED** [review]

**Comment by Tguntenaar** [comment]
@race-of-sloths score 8

**Pull request merged**
